test/e2e/compare.sh

```diff
@@ -3,7 +3,7 @@ set -e
 
 # Function to replace test duration with 0 and remove file paths
 remove_variables() {
-  echo "$1" | sed -E 's/\"duration\": [0-9\.]+/\"duration\": 0/g' | sed -E 's/\/.*\/test\//test\//g'
```

Comment:
Note: do not escape "." in a regexp class.  The backslash becomes part of the class rather than escaping ".".